packages/ui/src/views/repo/repo-files/file-review-error.tsx

```diff
@@ -16,13 +15,9 @@ export const FileReviewError = ({
       description={['Please try again or check your connection']}
       imageName="no-data-error"
       secondaryButton={{
-        label: (
-          <>
-            <IconV2 name="refresh" />
-            {t('views:repos.fileContent.noData.tryAgain', 'Try again')}
-          </>
-        ),
-        props: { onClick: onButtonClick }
+        icon: 'refresh',
+        label: t('views:repos.fileContent.noData.tryAgain', 'Try again'),
+        onClick: onButtonClick
```

Comment:
Much better now 👏 

Please ensure `label` as `string` should suffice all use case what we have.